### PR TITLE
Feature/create layout for displayed question

### PR DESCRIPTION
**Pull request by khlopkov**

### CODE REVIEW

**Comment by sponkrashin** [comment]
Are you sure you really need to use one more package, to add a bit more complexity just because of an issue that can be solved with `flex: ...` css attribute?

**Comment by khlopkov** [comment]
> 
> 
> Are you sure you really need to use one more package, to add a bit more complexity just because of an issue that can be solved with `flex: ...` css attribute?

I told with @santee about flex and he mentioned angular flex, so i decided to use it. 
What do you mean about complexity? I thought that angular flex make it less complex

**Comment by sponkrashin** [comment]
CSS's `flex` is a standard, `@angular/flex-layout` is just an external library that can be changed any time. `flex` layout is well-known for developers, but a developer will have to spend some time to understand what is going on with `@angular/flex-layout`. And from my point of view, it doesn't bring any significant advantages, it just encapsulates, for instance, `flex-direction: column` under `fxLayout="column"` directive. Also, for me it is harder to understand, what `fxFlex="8 6 0" fxFlex.xs="5 5 50"` means. I had to refer to the documentation.

File: Arcadia.Ask/ClientApp/src/app/dashboard/main/main.component.html [hunk 1]
```diff
@@ -1,7 +1,29 @@
-<app-questions-list 
-  [editable]="canEditQuestions| async"
-  [votingAvailable]="canVoteQuestions | async"
+<div class="content" 
+  fxLayout="row" fxLayout.xs="column"
```
Comment (sponkrashin) [inline Arcadia.Ask/ClientApp/src/app/dashboard/main/main.component.html]: Maybe it really makes sense. Ok, let it be.

File: Arcadia.Ask/ClientApp/src/app/dashboard/main/main.component.html [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
+  <app-displayed-question
+    fxFlex="8 6 0" fxFlex.xs="5 5 50"
+    [editable]="canEditQuestions| async"
+    [votingAvailable]="canVoteQuestions | async"
+  >
+  </app-displayed-question>
+
+  <div class="question-sidebar" 
+    fxFlex="3 2 0" fxFlex.xs="5 5 50"
+    fxLayout="column" fxLayoutAlign="start stratch"
```
Comment (sponkrashin) [inline Arcadia.Ask/ClientApp/src/app/dashboard/main/main.component.html]: Probably typo here.

```suggestion
    fxLayout="column" fxLayoutAlign="start stretch"
```

File: Arcadia.Ask/ClientApp/src/app/questions/displayed-question/displayed-question.component.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.question-text {
+  overflow-y: auto;
+}
+
+.question-text > h1 {
+  overflow-wrap: anywhere;
```
Comment (sponkrashin) [inline Arcadia.Ask/ClientApp/src/app/questions/displayed-question/displayed-question.component.css]: Did you consider, that container will behave as the following when some long words occurred?
image

**Comment by sponkrashin** [comment]
Also I don't like inline styles. And this library adds some kind of them. The best practice is to separate code from representation, and this library violates this rule.

File: Arcadia.Ask/ClientApp/src/app/questions/displayed-question/displayed-question.component.css (hunk 3 above)
Comment (khlopkov) [reply Arcadia.Ask/ClientApp/src/app/questions/displayed-question/displayed-question.component.css]: But when I input long word in displayed question it displays correctly. But with `overflow-wrap: break-word;` incorrect

- `overflow-wrap: anywhere`
overflow-wrap: anywhere
big overflow-wrap: anywhere
- `overflow-wrap: break-word`
overflow-wrap: break-word
overflow-wrap: break-word big

Comment (sponkrashin) [reply Arcadia.Ask/ClientApp/src/app/questions/displayed-question/displayed-question.component.css]: Ok then.

**Pull request merged**